interactions/client/bot.py

```diff
@@ -507,13 +507,39 @@ async def wait_until_ready(self) -> None:
         """
         await self._websocket.wait_until_ready()
 
+    async def __get_all_guilds(self) -> List[dict]:
```

Comment:
Is there a reason why this method needs two prefixed underscores? It's not something that could mess with the bot is mismanaged, so at most it should only have one underscore

Reply:
good point, fixed